test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/TaskListControllerSpec.scala

```diff
@@ -2366,7 +2366,7 @@ class TaskListControllerSpec
             sample[DraftSingleMixedUseDisposalReturn].copy(
               triageAnswers = sample[CompleteSingleDisposalTriageAnswers]
                 .copy(individualUserType = None),
-              examplePropertyDetailsAnswers = Some(sample[CompleteExamplePropertyDetailsAnswers]),
+              //examplePropertyDetailsAnswers = Some(sample[CompleteExamplePropertyDetailsAnswers]),
```

Comment:
you can remove these commented out lines if they're not needed - there's a few in this file